### PR TITLE
feat: Use Cloudinary for images

### PR DESCRIPTION
**Pull request by glokta1**
1. Most of the work is done. Just need some help in adding the Cloudinary image link to the form data because I suck at React. I think I've done it incorrectly and it doesn't get stored in the `image` field under `mentorImg`
2. After merging this, we can remove the static rendering from the mentor list and mentor profile pages in a separate PR

Fixes: https://github.com/anmode/grabtern-backend/issues/2

### CODE REVIEW

File: components/MentorRegistration/MentorFormRegistration.js [hunk 1]
```diff
@@ -114,10 +114,31 @@ export default function MentorForm() {
...
+
   const handleUploadImageChange = async (e) => {
     const file = e.target.files[0];
-    const base64 = await convertBase64(file);
-    setFormData({ ...formData, mentorImg: { name: file.name, image: base64 } });
+    const fileData = await convertBase64(file);
+    console.log("The uploaded image is: ", file);
+    const imgLink = await uploadToCloudinary(fileData);
+    console.log(imgLink);
+    setFormData({
```
Comment (glokta1) [inline components/MentorRegistration/MentorFormRegistration.js:137]: Need help on this line specifically @Illaf @Riyazul555

**Comment by glokta1** [comment]
image
something's going wrong on this line

**Comment by anmode** [comment]
@glokta1 baaki images ko bhi upload krdein? kya hum.
ek aur cheej alag alag folder create hoskte h ?

**Comment by glokta1** [comment]
Should we make the demo CDN active or is it fine like this? @anmode

**Comment by anmode** [comment]
yess we should make it active

**Pull request merged**